examples/Chainweb/Simulate/Contracts/CryptoCritters.hs

```diff
@@ -7,7 +7,7 @@
 module Chainweb.Simulate.Contracts.CryptoCritters where
```

Comment:
As discussed can we ditch the extra contracts and just go with coin-contract stuff and something like simple payments? We don't want to maintain any more pact code than totally necessary.

Reply:
Should we also keep `HelloWorld`?